srcpkgs/xdg-desktop-portal-wlr/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'xdg-desktop-portal-wlr'
+pkgname=xdg-desktop-portal-wlr
+version=0.1.0
+revision=1
+build_style=meson
+configure_args="-Dsystemd=disabled"
+hostmakedepends="pkg-config wayland-devel"
+makedepends="elogind-devel pipewire-devel wayland-devel wayland-protocols"
+short_desc="Backend implementation for xdg-desktop-portal using wlroots"
+maintainer="Felix Kleinschmidt <fk@bsrd.io>"
+license="MIT"
+homepage="https://github.com/emersion/xdg-desktop-portal-wlr"
+distfiles="https://github.com/emersion/xdg-desktop-portal-wlr/archive/v$version.tar.gz"
```

Comment:
`distfiles="https://github.com/emersion/${pkgname}/archive/v${version}.tar.gz"`

Reply:
Please don't. Using variables (besides of $version, that is necessity) in url makes it harder to copy from template and use, and leaks to other data sets (search for $ in https://alpha.de.repo.voidlinux.org/void-updates/void-updates.txt ).
Also $pkgname is best when used as *package name*, ie. where it would be still used, if preferred pkgname was taken/forbidden.

Reply:
If you want to make the line smaller, `distfiles="${homepage}/archive/...` is valid

Reply:
$homepage have same downside of obfuscating real url.

Reply:
`distfiles="https://github.com/emersion/xdg-desktop-portal-wlr/archive/v${version}.tar.gz"` would be enough.